spec/features/review_card_spec.rb

```diff
@@ -1,12 +1,19 @@
 require "rails_helper"
 require "support/login"
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.